Orange/base.py

```diff
@@ -27,11 +27,15 @@ def __init__(self, preprocessors=None):
         self.preprocessors = list(preprocessors)
 
     def fit(self, X, Y, W=None):
-        raise NotImplementedError(
-            "Descendants of Learner must overload method fit")
+        raise SystemError(
```

Comment:
Based on documentation of [SystemError](https://docs.python.org/3.5/library/exceptions.html#SystemError) and [NotImplementedError](https://docs.python.org/3.5/library/exceptions.html#NotImplementedError), the latter seems more suitable for this particular error.